.github/workflows/promote.yml

```diff
@@ -1,23 +1,18 @@
-name: promote
+name: Promote
 on:
   workflow_dispatch:
     inputs:
-      core-version:
-        description: The PatternFly core version
+      release-version:
+        description: The version of PatternFly to promote to.
```

Comment:
FWIW, the argument in this workflow is the @patternfly/patternfly package version which is updated where ever necessary in the various packages in the patternfly-react monorepo. So it _should_ be the same as the promoted version, but it isn't necessarily always.

Reply:
I understood that as well, but I thought `core-version` was a bit non-descriptive, I've changed this to `patternfly-version` now, with some additional description.